doc/configure-device.md

```diff
@@ -36,6 +36,98 @@ To debug Flutter apps on your Tizen device (watch or TV), you need to configure
+   $ sudo ./sd_fusing_rpi3.sh -d /dev/sdc --format
+   
+   Raspberry Pi downloader, version 2.1.0
+
+   Device: /dev/sdc
+
+   /dev/sdc will be formatted, Is it OK? [y/<n>]
+
+   ...
+
+   End /dev/sdc format
+   ```
+
+1. Download the Tizen OS image. Visit the links below and download two files for boot and platform. The file names end with `.tar.gz`. Choose Tizen 6.0 (arm) if you are not sure which one to download.
+
+   | Tizen version | Image type | Latest image |
+   |-|-|-|
+   | Tizen 5.5 | Pi 3 (arm) | [iot-boot-armv7l-rpi3](http://download.tizen.org/snapshots/tizen/5.5-unified/latest/images/standard/iot-boot-armv7l-rpi3)<br>[iot-headed-3parts-armv7l-rpi3](http://download.tizen.org/snapshots/tizen/5.5-unified/latest/images/standard/iot-headed-3parts-armv7l-rpi3) |
+   | Tizen 6.0 | Pi 3 (arm) | [iot-boot-armv7l-rpi3](http://download.tizen.org/snapshots/tizen/6.0-unified/latest/images/standard/iot-boot-armv7l-rpi3)<br>[iot-headed-3parts-armv7l-rpi](http://download.tizen.org/snapshots/tizen/6.0-unified/latest/images/standard/iot-headed-3parts-armv7l-rpi) |
+   | | Pi 3 (arm64) | [iot-boot-arm64-rpi3](http://download.tizen.org/snapshots/tizen/6.0-unified/latest/images/standard/iot-boot-arm64-rpi3)<br>[iot-headed-3parts-aarch64-rpi](http://download.tizen.org/snapshots/tizen/6.0-unified/latest/images/standard/iot-headed-3parts-aarch64-rpi) |
+   | | Pi 4 (arm) | [iot-boot-armv7l-rpi4](http://download.tizen.org/snapshots/tizen/6.0-unified/latest/images/standard/iot-boot-armv7l-rpi4)<br>[iot-headed-3parts-armv7l-rpi](http://download.tizen.org/snapshots/tizen/6.0-unified/latest/images/standard/iot-headed-3parts-armv7l-rpi) |
+   | | Pi 4 (arm64) | [iot-boot-arm64-rpi4](http://download.tizen.org/snapshots/tizen/6.0-unified/latest/images/standard/iot-boot-arm64-rpi4)<br>[iot-headed-3parts-aarch64-rpi](http://download.tizen.org/snapshots/tizen/6.0-unified/latest/images/standard/iot-headed-3parts-aarch64-rpi) |
+
+1. Flash the image to the SD card. For example:
+
+   ```sh
+   $ sudo ./sd_fusing_rpi3.sh -d /dev/sdc -b tizen-6.0-unified_20210512.1_iot-boot-arm64-rpi3.tar.gz \
+   tizen-6.0-unified_20210512.1_iot-headed-3parts-aarch64-rpi.tar.gz
+
+   Raspberry Pi downloader, version 2.1.0
+
+   Device: /dev/sdc
+   Fusing binary:
+
+   ...
+
+   [Fusing user.img (108 MiB)]
+   109051904 bytes (109 MB, 104 MiB) copied, 7 s, 15.1 MB/s
+   27+1 records in
+   27+1 records out
+   113410048 bytes (113 MB, 108 MiB) copied, 7.52632 s, 15.1 MB/s
+   resize2fs 1.45.5 (07-Jan-2020)
+   Resizing the filesystem on /dev/sdc5 to 6456832 (4k) blocks.
+   The filesystem on /dev/sdc5 is now 6456832 (4k) blocks long.
+   ```
+
+1. Done. Detach the SD card and insert to the Pi device.
+
+### Connect
+
+1. Connect the Pi device to the same network as your host PC using an Ethernet or Wi-Fi (built-in or extra USB dongle) connection.
+
+   Alternatively, you can use some extra hardware (USB/PWR splitter) to connect Pi directly to the host PC in OTG (USB slave) mode.
```

Comment:
I'd add a note, that when we connect a device over USB, we don't have to issue `sdb connect ...` - it may not be so obvious to devs new to Tizen ;)

```suggestion
  > Alternatively, you can use some extra hardware (USB/PWR splitter) to connect Pi directly to the host PC in OTG (USB slave) mode. The Tizen device will be ready to use just after plugging the USB - the next step should be skipped.
```

Reply:
Thank you!